charts/argo/templates/workflow-controller-config-map.yaml

```diff
@@ -67,3 +67,6 @@ data:
     {{- if .Values.controller.workflowDefaults }}
     workflowDefaults:
 {{ toYaml .Values.controller.workflowDefaults | indent 6 }}{{- end }}
+    {{- if .Values.controller.sso }}
```

Comment:
I suggest a `with` block too for consistency and to clean things up and indenting the toYaml to the correct place.

Reply:
done